src/components/calculators/Payslip.vue

```diff
@@ -139,10 +330,9 @@ export default {
         font-weight: 600;
         margin-right: 0;
         list-style: none;
-    }
-
-    .tabs-component-tab:not(:last-child) {
-        border-bottom: dotted 1px #ddd;
+        width: 33.3%;
```

Comment:
Usually best to give yourself a few more decimal places to make the browser divide it as accurately as it can (I usually use 4) 33.3333%